### PR TITLE
1.6 compatibility

### PR DESCRIPTION
**Pull request by judy2k**
Made some minor changes to allow 1.6 compatibility (at least source compatibility - I haven't tested against the 1.6 classpath). This gives us compatibility with any JVM from the past 10 years, which should be more than enough, and the cost is not enormous.

If we decide to move to 1.7+ (which is the past 5 years), we can make that decision further down the line.

### CODE REVIEW

**Comment by judy2k** [comment]
I've taken the testing out of Travis, so we will get compiler errors if we use 1.7+ features, but no guarantees that the code will run under 1.6.

> JDK 7 is backwards compatible, we think it’s time for all projects to start testing against JDK 7 first and JDK 6 if resources permit. (from Travis docs)

File: src/main/java/com/nexmo/messaging/sdk/callback/AbstractMOServlet.java [hunk 1]
```diff
@@ -165,10 +165,9 @@ private void handleRequest(HttpServletRequest request, HttpServletResponse respo
             this.consumer.execute(task);
 
             // immediately ack the receipt
-            try (PrintWriter out = response.getWriter()) {
```
Comment (ChrisGuzman) [inline src/main/java/com/nexmo/messaging/sdk/callback/AbstractMOServlet.java]: Why did you remove the `try` block here?

Comment (judy2k) [inline src/main/java/com/nexmo/messaging/sdk/callback/AbstractMOServlet.java]: The driver for the change was that the try (x) { blah; } syntax is 1.7+.

I should probably put in a try/catch. I wasn't aware the new syntax enforced a call to `close` in this case. Thanks for the prompt!

**Comment by ChrisGuzman** [comment]
@judy2k Does this need another review?

**Comment by judy2k** [comment]
@ChrisGuzman No, it needs a change, but it's low priority!

**Comment by judy2k** [comment]
We don't have a dependency on this, and the PR is stale, so closing.
